main.tf

```diff
@@ -248,10 +240,10 @@ resource "kubectl_manifest" "karpenter_node_class" {
     apiVersion: karpenter.k8s.aws/v1beta1
     kind: EC2NodeClass
     metadata:
-      name: default
+      name: truemark
```

Comment:
`truemark-amd64`

Reply:
this is nodeclass, nodepools have amd/arm64 suffix